### DISCUSSION TITLE
file attachment on a SOAP call

### DISCUSSION THREAD

**Discussion by sprklinginfo** [CodeDredd/laravel-soap#286] [Q&A]
Does the package support attaching multiple files with a SOAP call? read the doc and all issues and didn't find any related information. This is the first time I am working on calling an external SOAP API that asks for [SOAPAttachment](https://www.w3.org/TR/SOAP-attachments/). If you know how to do it with this package or use a different package, please let me know. Many thanks!!

**Comment by veewee** [comment]
Hello @sprklinginfo 

This isn't a very common operation which means it is not implemented in a lot of packages.
Ive did some investigations in the past and found 2 packages that implement this in PHP.

More info van be found here:
https://github.com/phpro/soap-client/discussions/357

I'm not sure how good those packages really are.

It's also something I want to eventually implement in the https://github.com/php-soap organization at some point, but did not got to yet. If it's available in there, it could eventually be activated in this package as well.

Good luck and feel free to let me know how things turn out for you :)
